src/JellyfishClient.ts

```diff
@@ -158,7 +159,10 @@ export interface Config<PeerMetadata> {
   signaling?: SignalingUrl;
 }
 
-export type TrackContext<TrackMetadata> = Omit<RawTrackContext, "metadata"> & { metadata: TrackMetadata };
+export type CreateConfig<PeerMetadata, TrackMetadata> = {
```

Comment:
Maybe `JellyfishClientConfig` would be more precise?

Reply:
We have two config objects: one for the `create` method and one for the `connect` method, so I named them "CreateConfig" and "ConnectConfig" respectively. "CreateConfig" is similar to a constructor config from "membrane-webrtc-js", but the one from "membrane-webrtc-js" is not exported. I don't think we should include Jellyfish as part of this type, since we only have one class exported from this repository.